.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,15 @@
+name: Build and deploy Jekyll site to GitHub Pages
```

Comment:
Do we need to move to Github actions?

I think that even without this file, the site would still be deployed in the same way it currently is

Reply:
There may be another way using the regular GitHub Jekyll build, but I haven't found one yet.

The issue is that we need Bootstrap sources to build the CSS, and I don't know how to dynamically import it with the regular method (other than copying it in our repo).

Problematic line is `@import "bootstrap";` in `assets/css/style.scss`

GitHub error msg:
> Your site is having problems building: Your SCSS file assets/css/style.scss has an error on line 2: File to import not found or unreadable: bootstrap. Load paths: assets/css /hoosegow/.bundle/ruby/2.7.0/gems/jekyll-theme-primer-0.5.4/_sass. For more information, see https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#invalid-sass-or-scss.

But GitHub Actions does give us additional benefits:

> **Control over gemset**
> * **Jekyll version** — Instead of using the currently enabled version at `3.9.0`, you can use any version of Jekyll you want. For example `4.2.0`, or point directly to the repository.
> * **Plugins** — You can use any Jekyll plugins irrespective of them being on the [supported versions](https://pages.github.com/versions/) list, even `*.rb` files placed in the `_plugins` directory of your site.
> * **Themes** — While using a custom theme is possible without Actions, it is now simpler.
>
> **Workflow Management**
> * **Customization** — By creating a workflow file to run Actions, you can specify custom build steps, use environment variables.
> * **Logging** — The build log is visible and can be tweaked to be verbose, so it is much easier to debug errors using Actions.